gitea/repo_file.go

```diff
@@ -13,3 +13,89 @@ import (
 func (c *Client) GetFile(user, repo, ref, tree string) ([]byte, error) {
 	return c.getResponse("GET", fmt.Sprintf("/repos/%s/%s/raw/%s/%s", user, repo, ref, tree), nil, nil)
 }
+
+// IdentityOptions for a person's identity like an author or committer
+type IdentityOptions struct {
+	Name string `json:"name" binding:"MaxSize(100)"`
+	// swagger:strfmt email
+	Email string `json:"email" binding:"MaxSize(254)"`
+}
+
+// FileOptions options for all file APIs
+type FileOptions struct {
+	Message       string          `json:"message"" binding:"Required"`
```

Comment:
I think the following will fix the CI fail
```suggestion
	Message       string          `json:"message" binding:"Required"`
```

Reply:
Thanks for the find!